scripts/assets/entry-alpine.sh

```diff
@@ -1,18 +1,29 @@
 #!/bin/bash
 
+hostname "$HOSTNAME" &> /dev/null
+if [[ $? == 0 ]]; then
```

Comment:
@nghiant2710 what is the mechanism we are using to determine if the container is privileged or not?

Reply:
Run hostname command and check its exit code is the mechanism we use here